src/dbnode/storage/index/options.go

```diff
@@ -212,8 +212,8 @@ func (o *opts) Validate() error {
 	if o.postingsListCache == nil {
 		return errPostingsListCacheUnspecified
 	}
-	if o.queryStats == nil {
-		return errOptionsQueryStatsUnspecified
+	if o.queryLimits == nil {
```

Comment:
nit: considering this has a default, may not be worth having a validate for it?

Reply:
Yeah that is consistent w/ the other options here